SConstruct

```diff
@@ -445,6 +445,9 @@ if not env_base["deprecated"]:
 if env_base["precision"] == "double":
     env_base.Append(CPPDEFINES=["REAL_T_IS_DOUBLE"])
 
+if env_base["platform"] == "linuxbsd" and not env_base["x11"]:
+    env_base["opengl3"] = False
```

Comment:
I don't see a reason to disable OpenGL support when not using X11 _a priori_.

Wayland can use OpenGL too (requires some changes to use EGL instead of GLX, but generally speaking OpenGL and X11 don't have to be tied).

I think this was meant to fix a build failure for the OpenGL Xlib extension in OpenXR - that's the thing which should be disabled, not the whole OpenGL renderer.